dataflow/src/node/debug.rs

```diff
@@ -40,33 +50,42 @@ impl Node {
             },
             None => format!("{} / -", idx.index()),
         };
+
         match self.inner {
             NodeType::Source => s.push_str("(source)"),
             NodeType::Dropped => s.push_str("✗"),
-            NodeType::Ingress => s.push_str(&format!("{{ {} | (ingress) }}", addr)),
-            NodeType::Egress { .. } => s.push_str(&format!("{{ {} | (egress) }}", addr)),
+            NodeType::Ingress => s.push_str(&format!(
+                "{{ {{ {} {} }} | (ingress) }}",
+                addr,
+                materialized
+            )),
+            NodeType::Egress { .. } => {
+                s.push_str(&format!("{{ {{ {} {} }} | (egress) }}", addr, materialized))
+            }
             NodeType::Sharder { .. } => s.push_str(&format!("{{ {} | (sharder) }}", addr)),
```

Comment:
Can a `Sharder` be materialized?

Reply:
No, a `Sharder` cannot be materialized afaik.